src/raygun.rum/vendor/web-vitals.vendor.js

```diff
@@ -289,7 +292,7 @@
         // best we can do until an API is available to support querying past
         // visibilityState.
         {
-          firstHiddenTime = self.webVitals.firstHiddenTime;
+          firstHiddenTime = window.webVitals ? window.webVitals.firstHiddenTime : initHiddenTime();
```

Comment:
I think this should continue to use `self` to access the `webVitals` object. e.g.

```
!!self.webVitals ? self.webVitals.firstHiddenTime : initHiddenTime();
```

This ensures it will still work in Web Workers